### ISSUE TITLE
Kitti dataset evaluation results

### ISSUE THREAD

**Issue by Majiawei** [fizyr/keras-retinanet#1384]
Retinanet uses resnet50 as backbone to train Kitti data sets. How much is the result of Moderate, Easy and Hard on car, pedestrian and cyclist? Has anyone trained?

**Comment by Majiawei** [comment]
This is my training result. The test set is half of the training set. Is the test result normal？
car_detection AP: 96.442238 86.703117 77.700768
pedestrian_detection AP: 69.517525 60.272072 52.395069
cyclist_detection AP: 76.390060 61.614601 61.041290
